kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -407,11 +424,6 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/props', '
 
     function createJobStatusLines(jobState, includeHistory = false) {
         if (!isValidJobStateObject(jobState)) {
-            // check whether EE2 sent { job_state: 'does_not_exist' }
-            // TODO: coerce the { job_state: 'does_not_exist' } into a standard format
-            if (jobState && jobState.job_state && jobState.job_state === 'does_not_exist') {
-                return jobNotFound;
-            }
```

Comment:
Get rid of this hack as it is no longer required -- `job_state: "does_not_exist"` should not come from the backend and will not be generated in the frontend.